flag.go

```diff
@@ -595,3 +596,27 @@ func withEnvHint(envVar, str string) string {
 	}
 	return str + envText
 }
+
+func stringifyFlag(f Flag) string {
+	fv := reflect.ValueOf(f)
+	placeholder, usage := unquoteUsage(fv.FieldByName("Usage").String())
+
+	defaultValueString := ""
+	if val := fv.FieldByName("Value"); val.IsValid() {
+		defaultValueString = fmt.Sprintf("%v", val.Interface())
+	}
+
+	formatFunc := fv.MethodByName("FormatValueHelp")
+	if formatFunc.IsValid() {
+		defaultValueString = formatFunc.Call([]reflect.Value{})[0].String()
```

Comment:
At the very least, we should assert the signature of this method before calling it.